php/example_code/lambda/Readme.md

```diff
@@ -0,0 +1,98 @@
+# AWS Lambda code examples for the SDK for PHP
+
+## Overview
+
+Shows how to use the AWS SDK for PHP to create, deploy, and invoke
+AWS Lambda functions. Learn to accomplish the following tasks
+
+*Lambda lets you run code without provisioning or managing servers. Upload your code
+and Lambda takes care of everything required to run and scale your code with high
+availability.*
```

Comment:
New paragraph (service blurb from [Lambda landing page](https://docs.aws.amazon.com/lambda/)):

*With AWS Lambda, you can run code without provisioning or managing servers. You pay only for the compute time that you consume—there's no charge when your code isn't running.*

Reply:
replaced